### DISCUSSION TITLE
Move all the old `documentation` to README files?

### DISCUSSION THREAD

**Discussion by ghost** [leokhoa/laragon#439] [Ideas]
Does anyone have all the old documentation topics of the forum? Or maybe a cache of all these topics ?
All these links in  the Laragon app link to a forum that's currently down.

Such as 
- How to add another PHP version
- How to add another MYSQL version
- How to manage Quick Apps 
- etc

Personally I know how to do this, but maybe for starters it would be nice to have this documentation again. Having them hosted for free on Github would be beneficial. 

Personally I needed it for the following: https://github.com/laravel/horizon/issues/78#issuecomment-326681677

Trying to run `laravel/horizon` on a local Windows machine, and I saw that @leokhoa possibly ported the `pcntl` extension. Which is needed for Horizon, otherwise I might have to go for a Homestead installation.

Maybe this can be added to a documentation folder for the time being so other people can work on the readme files ?

EDIT:
Added a basic README file on how to add another version of PHP https://github.com/leokhoa/laragon/pull/440

**Comment by ghost** [comment]
Actually found the download link for de windows port of `pcntl` using Wayback Machine.

https://sourceforge.net/projects/laragon/files/addon/ext/7.1/php_pcntl-0.1-7.1-ts-vc14-x64.zip/download

But doesn't seem to work.

![image](https://user-images.githubusercontent.com/25492573/224967475-6a09f197-d936-4f62-ba03-0ef747599f78.png)

**Reply by Pen-y-Fan** [reply]
My understanding is pcntl is not compatible with Windows:  [How can I install the PCNTL extension on Windows?](https://stackoverflow.com/questions/30045085/how-can-i-install-the-pcntl-extension-on-windows). 

An extension written for PHP 7.1 will not work with PHP 8.2

Checking out a possible workaround: https://github.com/laravel/horizon/issues/154#issuecomment-366712260

**Reply by ghost** [reply]
> My understanding is pcntl is not compatible with Windows: [How can I install the PCNTL extension on Windows?](https://stackoverflow.com/questions/30045085/how-can-i-install-the-pcntl-extension-on-windows).
> 
> An extension written for PHP 7.1 will not work with PHP 8.2
> 
> Checking out a possible workaround: [laravel/horizon#154 (comment)](https://github.com/laravel/horizon/issues/154#issuecomment-366712260)

Ah somehow I overlooked that it wqs a 7.1 extension. I knew that PCNTL isn’t compatible with Windows, but in the comment I linked they said:

> The very talented developer of Laragon has ported the extension (more or less, it's still experimental at this point) to windows, you can get it here: https://forum.laragon.org/topic/593/how-to-add-pcntl-extension-windows

Which is where I got it from.
Haven’t tried the ext-pcntl ext-posix with composer yet so I’ll give that a shot later.
